.github/workflows/pull_request.yml

```diff
@@ -100,7 +100,7 @@ jobs:
           branch_name='preview-${{ github.event.pull_request.number }}'
           git add --force resources/spell_check_results.tsv
           git commit -m 'Add spell check file' || echo "No changes to commit"
-          git pull --set-upstream origin $branch_name --allow-unrelated-histories
+          git pull --set-upstream origin $branch_name --allow-unrelated-histories --strategy-option=theirs
```

Comment:
This helps resolve merge conflict issues

Reply:
oh yay!